automated install/basic-install.sh

```diff
@@ -1150,6 +1156,66 @@ if [[ "${reconfigure}" == true ]]; then
     fi
 }
 
+FTLdownload() {
+  # Download suitable FTL binary
+  echo ":::"
+  echo "::: Downloading latest version of FTL..."
+
+  local machine=$(uname -m)
+
+  echo $machine
+
+  if [[ $machine == arm* ]]; then
+    # ARM
+    if [ -f "/lib/ld-linux-aarch64.so.1" ]; then
+      echo ":::  Detected ARM-aarch64 architecture"
+      binary="pihole-FTL-aarch64-linux-gnu"
+    elif [ -f "/lib/ld-linux-armhf.so.3" ]; then
+      echo ":::  Detected ARM-hf architecture"
+      binary="pihole-FTL-arm-linux-gnueabihf"
+    else
+      echo ":::  Detected ARM architecture"
+      binary="pihole-FTL-arm-linux-gnueabi"
+    fi
+  elif [[ $machine == x86_64 ]]; then
+    # 64bit
+    echo ":::  Detected x86_64 architecture"
+    binary="pihole-FTL-linux-x86_64"
+  else
+    # Something else - we try to use 32bit executable and warn the user
+    if [[ ! $machine == i686 ]]; then
+      echo ":::  Not able to detect architecture (unknown: ${machine}), trying 32bit executable"
+      echo ":::  Contact Pi-hole support if you experience problems (like FTL not running)"
+    fi
+    binary="pihole-FTL-linux-x86_32"
```

Comment:
How about check for 32 bit first (and say that it's detected), but if it's still not known default to 32 bit and show the message. Then the 32 bit users will also have the `:::  Detected x86_64 architecture` line but unknown ones will still function the same.